DOCS/interface-changes/vapoursynth-api-v4.txt.txt

```diff
@@ -0,0 +1 @@
+Bump dependency of VapourSynth to utilize its API version 4. Some functions and plugins are changed or removed. For details, refer to VapourSynth documentation http://www.vapoursynth.com/2021/09/r55-audio-support-and-improved-performance/ and https://github.com/vapoursynth/vapoursynth/blob/R68/APIV4%20changes.txt
```

Comment:
Does it really affect mpv?

Reply:
> Does it really affect mpv?
> 
> Currently on Windows users can use any version of vs with mpv. Do you mean this PR block users to use the R55- ?

I feel like that should be fine. R55 came out 3 years ago. If you're upgrading to the latest MPV builds, you're probably also using the latest VapourSynth too.

Reply:
> I feel like that should be fine. R55 came out 3 years ago. If you're upgrading to the latest MPV builds, you're probably also using the latest VapourSynth too.

You totally misunderstood it.
IF it doesn't require the minimum version of VS: There is no need to add things in interface-changes. Since it wouldn't change anything for those users.

Reply:
I thought the concern you raised in the first comment was that, for those users who are extremely stubborn to upgrade their VapourSynth, before this change they can grab the latest mpv and keep their VapourSynth R54, and still use all their scripts. After this change if they keep using R54, mpv will simply not initialize the filter. They are force to download VapourSynth R56 or later.

Also, before R55, there are [some VapourSynth filters](https://github.com/vapoursynth/vapoursynth/blob/R54/doc/plugins/misc.rst) that are no longer packed in the standard VapourSynth installation. After these users upgrade, they have to install them separately.

All of these affect users more than just "install the latest mpv". They have to do some collateral tasks to restore the previous behavior. I think it is good let user know. If `interface-changes` is not for this purpose, could you advise a better good place to go?

Reply:
> You totally misunderstood it.

@CrendKing That's talking to another one. And he seems to have deleted the posts.
I'm fine with the current commit.